.rubocop.yml

```diff
@@ -5,19 +5,19 @@ AllCops:
   TargetRubyVersion: 2.4.0
 
 Metrics/LineLength:
-  Max: 100
+  Enabled: false
```

Comment:
Why do we need lines longer than this?

Reply:
I've set the max line length to 80 characters. Fixed in 64d765b.